### DISCUSSION TITLE
CSV export errors caused by comma

### DISCUSSION THREAD

**Discussion by uzemelteteshj** [Budibase/budibase#11189] [Help] [answered]
If the user enters a comma in the text field then when it is entered into excel it will not tag in the right place.
How would you solve this?

**Comment by R2bEEaton** [comment, marked as answer]
Hello!

To fix this you could either fix the commas or use a different separator. But regardless, if the user enters the separator it will cause the same problem. So in that case, a good solution would be disallowing commas in the field.

Click on your text field and press "Configure Validation" to add a rule like the following:

![image](https://github.com/Budibase/budibase/assets/34921506/8dc970ee-7872-450d-a9d5-48e3315d5333)

The Regular Expression `.*[,].*` will match any text that includes a comma, and since it is required that the text not match the Regex, it will show an error to the user when they try to submit with a comma. For submitting, make sure you add a Validate Form action on your submit button (or however you submit).

![image](https://github.com/Budibase/budibase/assets/34921506/d45a9c8c-ea35-4bd7-b342-9daa68d11043)

Hope this helps! Let me know if you have any further questions. Otherwise, make sure to go ahead and accept this answer so your question can help future users! 😃

**Reply by uzemelteteshj** [reply]
It's work, thank you!!! :)
